src/lib/action.rs

```diff
@@ -147,11 +202,47 @@ impl<T: Clone> ApplyAction<T> for Update<T> {
     }
 }
 
-impl<T: Clone> ApplyAction<T> for ActionList<T> {
+impl<T: Clone> Default for EditScript<T> {
```

Comment:
I'm not sure `Default` is a great name (it's just so generic). Is there a better alternative?